### ISSUE TITLE
Test suite fails with Unicode::Casing

### ISSUE THREAD

**Issue by eserte** [briandfoy/roman-unicode#2]
See subject --- if Unicode::Casing is installed, then lowercase.t fails:
```
#   Failed test 'ↂↈ turns into ((|))(((|)))'
#   at t/lowercase.t line 48.
#          got: 'ↂↈ'
#     expected: '((|))(((|)))'

#   Failed test 'ↂↈⅯↂ turns into ((|))(((|)))ⅿ((|))'
#   at t/lowercase.t line 48.
#          got: 'ↂↈⅿↂ'
#     expected: '((|))(((|)))ⅿ((|))'

#   Failed test 'ↂↈⅯↂⅤⅠⅠ turns into ((|))(((|)))ⅿ((|))ⅴⅰⅰ'
#   at t/lowercase.t line 48.
#          got: 'ↂↈⅿↂⅴⅰⅰ'
#     expected: '((|))(((|)))ⅿ((|))ⅴⅰⅰ'

#   Failed test 'ↈↈ turns into (((|)))(((|)))'
#   at t/lowercase.t line 48.
#          got: 'ↈↈ'
#     expected: '(((|)))(((|)))'

#   Failed test 'ↈↈↈ turns into (((|)))(((|)))(((|)))'
#   at t/lowercase.t line 48.
#          got: 'ↈↈↈ'
#     expected: '(((|)))(((|)))(((|)))'
# Looks like you failed 5 tests of 19.
t/lowercase.t ..... 
Dubious, test returned 5 (wstat 1280, 0x500)
Failed 5/19 subtests 
```

**Comment by briandfoy** [comment]
Yeah, I'm trying to figure this out. Thanks,

**Comment by briandfoy** [comment]
This should be fixed in 1.033. Thanks,
